app/Filament/Resources/GameResource/RelationManagers/AchievementsRelationManager.php

```diff
@@ -219,7 +219,7 @@ public function table(Table $table): Table
             ->recordUrl(
                 fn (Achievement $record): string => route('filament.admin.resources.achievements.view', ['record' => $record])
             )
-            ->paginated([50, 100, 'all'])
+            ->paginated([50, 100, 150])
```

Comment:
Made an adjustment to the cap here so Livewire payload sizes don't infinitely scale.